Wireframe/index.html

```diff
@@ -1,13 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
The trailing slash has no effect here.
```suggestion
 <meta name="viewport" content="width=device-width, initial-scale=1.0" >